packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {
+  css,
+  autobind,
+  BaseComponent
+} from '../../Utilities';
+import { IOverflowSetProps } from './OverflowSet.Props';
+import { IconButton } from '../../Button';
+import { FocusZone, FocusZoneDirection } from '../../FocusZone';
+
+const styles: any = require('./OverflowSet.scss');
+
+export class OverflowSet extends BaseComponent<IOverflowSetProps, null> {
+
+  constructor(props: IOverflowSetProps) {
+    super(props);
+  }
+
+  public render() {
+    let { items, overflowItems } = this.props;
+    return (
+      <FocusZone direction={ FocusZoneDirection.horizontal } role='menubar' >
+        <div className={ css('ms-OverflowSet', styles.root) } >
```

Comment:
You can put the className on the FocusZone, which renders a div, to avoid the extra unnecessary div.